src/config/settings/base.py

```diff
@@ -139,6 +156,7 @@
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
     "allauth.account.middleware.AccountMiddleware",
+    "machina.apps.forum_permission.middleware.ForumPermissionMiddleware",
```

Comment:
a comment / link to the docs would be good for why this middleware is required.
If eventually django machina changes how it runs (perhaps dropping this middleware syntax) and thus updates its documentation, it might be hard to find justification for this. 
By including a link you then make this lines purpose as obvious and accessible as possible. 
You can look at the code below with links to django documentation for an example of what this could look like.

Reply:
Comments/links have been added.